019-ThisOldDataWarehouse/Student/Challenges/Challenge2/README.md

```diff
@@ -6,53 +6,55 @@
-The objective of this challenge is to build a Data Lake with Azure Data Lake Store (ADLS) Gen 2.  The Data Lake will be a staging area where all our source system data files reside. We need to ensure this Data Lake is well organized and doesn't turn into a swamp. This challenge will help us organize the folder sturcture and setup security to prevent unauthorized access.  Lastly, we will extract data from the WWI OLTP platform and store it in the Data Lake.  The OLTP platform is on-premise so you will need to build a hybrid archtiecture to integrate it into Azure.  Keep in mind that the pipeline that you build will become the EXTRACT portion of the new E-L-T process. The first requirement is to build a functional POC that is able to move a single dataset to the new ADLS Gen 2 data lake. Ideally, it would be nice to make the process table driven so that new pipelines do not need to be created for each additional table that needs to be copied. (Optional, sharing to give insights on end-state.)
-
-Note: This challenge is intended to build upon challenge 1, and you should try to reuse content wherever possible.
+The objective of this challenge is to build a Data Lake with Synapse Analytics or Azure Data Lake Store (ADLS) Gen 2.  The Data Lake will be a staging area where all our source system data files reside. We need to ensure this Data Lake is well organized and doesn't turn into a swamp. This challenge will help us organize the folder sturcture and setup security to prevent unauthorized access.  Lastly, we will extract data from the WWI OLTP platform and store it in the Data Lake.  The OLTP platform is on-premise so you will need to build a hybrid archtiecture to integrate it into Azure.  Keep in mind that the pipeline that you build will become the EXTRACT portion of the new E-L-T process. The first requirement is to build a functional POC that is able to move a single dataset to the new ADLS Gen 2 data lake. Ideally, it would be nice to make the process table driven so that new pipelines do not need to be created for each additional table that needs to be copied. (Optional, sharing to give insights on end-state.)
 
 ## Setup
 Prior to starting this challenge, you should ensure that there are changes in the City data captured from Wide World Importers OLTP Database.  Execute the script below to insert/change data in the source, and update necessary configuration values.
 
 1. Execute queries below in the Wide World Importers Database to update 10 existing records and insert 1 new record. 
-~~~~
-UPDATE T
-SET [LatestRecordedPopulation] = LatestRecordedPopulation + 1000
-FROM (SELECT TOP 10 * from [Application].[Cities]) T
-
-INSERT INTO [Application].[Cities]
-	(
-        [CityName]
-        ,[StateProvinceID]
-        ,[Location]
-        ,[LatestRecordedPopulation]
-        ,[LastEditedBy]
-	)
-    VALUES
-    (
-		'NewCity' + CONVERT(char(19), getdate(), 121)
-        ,1
-        ,NULL
-        , 1000
-        ,1
-	)
-;
+
+        UPDATE T
```

Comment:
add a code block, here an all the rest below.

Reply:
Updated code blocks